client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -166,9 +210,16 @@ const PluginsBrowser = ( {
 				id: 'plugins-search',
 			} );
 		}
+		if ( category ) {
+			items.push( {
+				label: translateCategoryTitle( { category, translate } ),
+				href: `/plugins/${ category }/${ siteSlug || '' }`,
+				id: 'category',
+			} );
+		}
 
 		dispatch( updateBreadcrumbs( items ) );
-	}, [ siteSlug, search ] );
+	}, [ siteSlug, search, category ] );
```

Comment:
I'm seeing a dependency warning here if we want to fix it, but it's probably an issue in trunk:

```suggestion
	}, [ siteSlug, search, category, dispatch, translate ] );
```